.github/workflows/releaseConfigurator.yaml

```diff
@@ -0,0 +1,64 @@
+name: Configurator Release
+
+# This workflow manually triggers the automated process of release based on the changelog.
+# Is recommended to run `make release-changelog` before to see an example of what is going to be generated.
+
+# IMPORTANT No matter the branch selected when triggered, the workflow will always execute on 'main' branch.
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    name: Create release
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.version.outputs.next-version }}
+      image-tag: ${{ steps.image-tag.outputs.tag }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: main
+      - uses: newrelic/release-toolkit/validate-markdown@v1
+      - uses: newrelic/release-toolkit/generate-yaml@v1
+        with:
+          excluded-dirs: charts,.github
+      - uses: newrelic/release-toolkit/is-held@v1
+      - uses: newrelic/release-toolkit/link-dependencies@v1
+        with:
+          dictionary: .github/rt-dictionary.yaml
+      - uses: newrelic/release-toolkit/next-version@v1
+        id: version
+      - id: image-tag
+        run: |
+          export IMAGE_TAG=$(echo "${{ steps.version.outputs.next-version }}" | sed 's/^v//')
+          IMAGE_TAG+="-pre"
+          echo ::set-output name=tag::$IMAGE_TAG
```

Comment:
I am a bit confused, aren't we always adding "-pre" No matter if it is a prerelease or not?

Reply:
ups! yes i need to remove this , thanks!